includes/modules/pages/advanced_search/header_php.php

```diff
@@ -24,3 +24,8 @@
   $sData['pfrom'] =    (isset($_GET['pfrom']) ? zen_output_string($_GET['pfrom']) : '');
   $sData['pto'] =      (isset($_GET['pto'])   ? zen_output_string($_GET['pto']) : '');
 
+  // check manufacturers 
+  $mfr_query = $db->Execute("SELECT count(*) AS count FROM " . TABLE_MANUFACTURERS); 
```

Comment:
It's more performant to `SELECT id .... LIMIT 1` and check for `->EOF` than to look through all the thousands of mfgs in the db just to count them.